library/lua/argparse.lua

```diff
@@ -196,4 +196,17 @@ function coords(arg, arg_name, skip_validation)
     return pos
 end
 
+function boolean(arg, arg_name)
+    local toBool={["true"]=true,["yes"]=true,["y"]=true,["on"]=true,["1"]=true,
+                  ["false"]=false,["no"]=false,["n"]=false,["off"]=false,["0"]=false}
```

Comment:
might as well declare this above the function so it doesn't get needlessly reevaluated on every call